### PR TITLE
Fix qt crash after 4930

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: src/llmq/instantsend.cpp [hunk 1]
```diff
@@ -1737,7 +1737,7 @@ bool CInstantSendManager::IsInstantSendMempoolSigningEnabled() const
 
 bool CInstantSendManager::RejectConflictingBlocks() const
 {
-    if (!masternodeSync->IsBlockchainSynced()) {
+    if (masternodeSync == nullptr || !masternodeSync->IsBlockchainSynced()) {
```
Comment (kittywhiskers) [inline src/llmq/instantsend.cpp:1740]: Is the whole expression evaluated before execution or is only the first valid expression evaluated?

Comment (PastaPastaPasta) [reply src/llmq/instantsend.cpp:1740]: it'll short circuit, so if the first part is true, then we will return false

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Review by kittywhiskers — APPROVED** [review]
utACK

**Pull request merged**